src/resmom/start_exec.c

```diff
@@ -5927,6 +5927,52 @@ job_nodes(struct job *pjob)
 	return job_nodes_inner(pjob, NULL);
 }
 
+/**
+ * @brief
+ * 	Resume multinode job after one or more sisters has been restarted
+ *
+ * @param[in] pjob - job pointer
+ *
+ * @return	Void
+ *
+ */
+
+void resume_multinode(job *pjob)
+{
+	if (pjob->ji_hosts == NULL)
+		return;
+
+	int com = IM_JOIN_RECOV_JOB;
+	hnodent *np = NULL;
+	eventent *ep = NULL;
+	int i;
+	for(i = 1; i < pjob->ji_numnodes; i++) {
+		np = &pjob->ji_hosts[i];
+
+		if( i == 1 )
+			ep = event_alloc(pjob, com, -1, np, TM_NULL_EVENT, TM_NULL_TASK);
+		else
+			ep = event_dup(ep, pjob, np);
+
+		if (ep == NULL) {
+			exec_bail(pjob, JOB_EXEC_FAIL1, NULL);
+			return;
+		}
+
+		int stream = np->hn_stream;
+		im_compose(stream, pjob->ji_qs.ji_jobid,
+			pjob->ji_wattr[(int)JOB_ATR_Cookie].at_val.at_str,
```

Comment:
@zv0n sorry but one very small change: (as now things have changed in mainline about how we access job attributes using getters/setters):
Can you please repalce this line with `get_jattr_str(pjob, JOB_ATR_Cookie)`?